src/ctim/schemas/openc2_network.clj

```diff
@@ -0,0 +1,117 @@
+(ns ctia.schemas.openc2-network
+  (:require [schema.core :as s]
+            [ctia.schemas.openc2vocabularies :as openc2v]))
+
+
+
+
+(s/defschema BGPBlackhole
+  {:type (s/enum "BGPBlackhole")
+   :host s/Str
+   })
+
+(s/defschema DNSSinkhole
+  {:type (s/enum "DNSSinkhole")
+   :host s/Str
+   })
+
+(def Protocol
+  (s/enum "TCP"
+          "UDP"
+          "ICMP"
+          "Any"))
+
+(def ACLAction
+  (s/enum "ALERT" 
+          "DROP"
+          "DENY"
+          "LOG"
+          "PASS"
+          "REJECT"))
+
+(s/defschema TrafficType
+  {:protocol Protocol
+   :sourceAddress s/Str
```

Comment:
I think as a matter of general principle, key names should not be camel case.  In CTIM, our keys are intended to be written to JSON, and so we use the JSON style with the underscore, like so:

``` clj
:source_address s/Str
```